src/modules/poweraccent/PowerAccent.UI/App.xaml.cs

```diff
@@ -41,20 +41,22 @@ protected override void OnExit(ExitEventArgs e)
 
         protected virtual void Dispose(bool disposing)
         {
-            if (!disposedValue)
+            if (_disposed)
+            {
+                return;
+            }
+
+            if (disposing)
             {
                 _mutex?.Dispose();
                 _themeManager?.Dispose();
-
-                // TODO: free unmanaged resources (unmanaged objects) and override finalizer
-                // TODO: set large fields to null
-                disposedValue = true;
             }
+
+            _disposed = true;
```

Comment:
`_disposed = true` should be under if (disposing) condition.

Reply:
Mmm... Nice catch!
Looked into docs https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1063
and this is outside the if statement.
I guess this is fine since the method is called with true parameter. Any wonder @jaimecbernardo ?